CMSgov/design-system
# [NO_TICKET] Misc script bug fixes

### Summary
- Fix issue where images from `@cmsgov/design-system-docs` is not being copied to a child DS's `docs/dist` folder
- Refactor babel config handling
- Specify `setupTests.js` to be in `src` in a Child DS similar to `create-react-app`
- Update deps, added react testing library
- Add error logging to babel
- Add ability to resolve from source dir for react examples

### How to test
[child-design-system-example 2.zip](https://github.com/CMSgov/design-system/files/4800807/child-design-system-example.2.zip)

1. Build doc site in Child DS, ensure the USA banner images are working 
2. Run `yarn test` in Child DS to ensure tests are working, then try moving `setupFiles.js` to a different folder and see the tests don't work anymore.
3. Go to a component in the child DS source and cause a babel error (i.e., change the import path to be incorrect). Ensure the babel error is logged out when trying to build
